ludwig/schema/model_config.py

```diff
@@ -447,6 +456,15 @@ def _set_gbm_attributes(self, config_dict: ModelConfigDict) -> None:
             else:
                 raise ValidationError("GBM Models currently only support Binary, Category, and Number " "features")
 
+    def _set_validation_parameters(self):
```

Comment:
What if `trainer.validation_field` or `traner.validation_metric` is already set in the config?

We should make sure we have a test for that scenario as well.

Reply:
Done.